habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -853,21 +863,29 @@ class HabitatSimFisheyeSemanticSensorConfig(SimulatorFisheyeSensorConfig):
 @dataclass
 class HeadRGBSensorConfig(HabitatSimRGBSensorConfig):
     uuid: str = "robot_head_rgb"
+    width: int = 256
```

Comment:
If my memory is correct, there was a reason to keep 640x480 resolution until now. What changed ?

Reply:
I don't think there was any reason. All configs that use this sensor are 256x256.

Reply:
This is fine for Fetch, but once we transition to Stretch (or Spot), we should align this actual camera config on those robots.

Reply:
The reason was to match the real camera resolution to enable easier sim2real and  be unbiased to  methods that require higher resolution (ObjectDetectors, SLAM).

Reply:
Soooo, is it okay for us to change it here ?